app/views/spina/admin/registers/_form.html.haml

```diff
@@ -64,7 +64,9 @@
   - unless @register.new_record?
     .pull-right= link_to t('spina.permanently_delete'), spina.admin_register_path(@register), method: :delete, data: {confirm: t('spina.pages.delete_confirmation', subject: @register.name) }, class: 'button button-link button-danger'
 
-= content_for :head do
+= content_for :plugin_stylesheets do
+  = javascript_include_tag '//unpkg.com/accessible-typeahead@0.4.0', data: {turbolinks_track: true}
```

Comment:
would be nice if we could host this ourselves using our own build pipeline, but not a blocker.